tests/xlsx_data/1.out/xl/sharedStrings.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<sst xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" count="5" uniqueCount="5"><si><t>Left</t></si><si><t>Middle</t></si><si><t>Right</t></si><si><t>L</t></si><si><t>R</t></si></sst>
\ No newline at end of file
+<sst xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" count="6" uniqueCount="6"><si><t>Left</t></si><si><t>Middle</t></si><si><t>Right</t></si><si><t>L</t></si><si><t>R</t></si><si><t>No Totals</t></si></sst>
```

Comment:
Do we want whitespace?

Reply:
This is test output. It's raw data from an xlsx file.